src/docfx/lib/HashUtility.cs

```diff
@@ -14,12 +15,19 @@ public static class HashUtility
         /// </summary>
         /// <param name="input">The input string</param>
         public static string GetMd5String(this string input)
+            => GetMd5String(new MemoryStream(Encoding.UTF8.GetBytes(input)));
+
+        /// <summary>
+        /// Get md5 hash string from stream
+        /// </summary>
+        /// <param name="stream">The input stream</param>
+        public static string GetMd5String(Stream stream)
         {
 #pragma warning disable CA5351 //Not used for encryption
             using (var md5 = MD5.Create())
 #pragma warning restore CA5351
             {
-                return new Guid(md5.ComputeHash(Encoding.UTF8.GetBytes(input))).ToString();
+                return new Guid(md5.ComputeHash(stream)).ToString();
```

Comment:
>.ToString() [](start = 56, length = 11)

This method returns a 'Guid' like string with dashes.

For backward compatibility, this is needed for document id.
But for other hash strings, I'd like to have a hex string without dash that is more aligned with other systems.